examples/workbooks/ProductsAndReviews.ts

```diff
@@ -292,60 +292,58 @@ export const runRecordHooks =
  * Main
  */
 
+const ChainValidateField = FF.makeField<string, {validateFuncs:any[]}>(
+  FF.TextField({}),
+  {},
+  (mergedOptions, passedOptions) => {
+    const validate = (val:string) => {
+
+      const allMessages = _.map(passedOptions.validateFuncs, (x) => x(val))
+      const presentMessages = allMessages.filter(isNotNil)
+      return presentMessages
+    }
+    const consolidatedOptions = FF.mergeFieldOptions(mergedOptions, {validate  })
+    return new FF.Field(consolidatedOptions)
+  })
+
+
 const ReviewsSheet = new FF.Sheet(
     "Reviews",
     {
-        review_id: FF.TextField({
+      review_id: ChainValidateField({
             label: "Review Id",
             description: "The unique ID of the review.",
             unique: true,
-            validate: (value) => {
-                const ensureMaxLength = validateMaxLength(50)(value);
-                const ensureValidChars = validateRegex(/[\w-_]+/g)(value);
-
-                return runValidations(ensureMaxLength, ensureValidChars);
-            },
```

Comment:
These validate functions are repitious.  We combined them with the new argument named `validateFuncs`